securedrop/i18n_tool.py

```diff
@@ -40,10 +40,12 @@ class I18NTool(object):
         'fr_FR': {'name': 'French', 'desktop': 'fr', },
         'el': {'name': 'Greek', 'desktop': 'el', },
         'hi': {'name': 'Hindi', 'desktop': 'hi', },
+        'is': {'name': 'Icelandic', 'desktop': 'is', },
         'it_IT': {'name': 'Italian', 'desktop': 'it', },
         'nb_NO': {'name': 'Norwegian', 'desktop': 'nb_NO', },
         'nl': {'name': 'Dutch', 'desktop': 'nl', },
         'pt_BR': {'name': 'Portuguese, Brasil', 'desktop': 'pt_BR', },
+        'ro': {'name': 'Romanian', 'desktop': 'ro', },
```

Comment:
two other places we should update: 

docs here: https://github.com/freedomofpress/securedrop/blob/develop/docs/includes/l10n.txt
locales to test against in CI here: https://github.com/freedomofpress/securedrop/blob/develop/securedrop/bin/translation-test#L20

Reply:
@redshiftzero I am guessing both of those should be updated after we merge the translations. Otherwise, the commands and tests will fail.